.github/workflows/ci.yml

```
@@ -0,0 +1,75 @@
+name: ci
+
+"on":
+  push:
+    paths:
+      - ".github/workflows/ci.yml"
+      - "lib/**"
+      - "*.gemspec"
+      - "spec/**"
+      - "Rakefile"
+      - "Gemfile"
+      - ".rubocop.yml"
+  pull_request:
+    branches:
+      - master
+  create:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.7"
+        db:
+          - sqlite3
+          - mysql
+          - postgresql
+    env:
+      DB: ${{matrix.db}}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install package dependencies
+        run: "[ -e $APT_DEPS ] || sudo apt-get install -y --no-install-recommends $APT_DEPS"
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+          ruby-version: ${{matrix.ruby}}
+      - name: Run all tests
+        env:
+          HANAMI_DATABASE_USERNAME: root
+          HANAMI_DATABASE_PASSWORD: root
+          HANAMI_DATABASE_HOST: 127.0.0.1
+          HANAMI_DATABASE_NAME: hanami_model
+        run: script/ci
```

```suggestion     
      HANAMI_DATABASE_USERNAME: root
      HANAMI_DATABASE_PASSWORD: root
      HANAMI_DATABASE_HOST: 127.0.0.1
      HANAMI_DATABASE_NAME: hanami_model
    steps:
      - uses: actions/checkout@v1
      - name: Install package dependencies
        run: "[ -e $APT_DEPS ] || sudo apt-get install -y --no-install-recommends $APT_DEPS"
      - name: Set up Ruby
        uses: ruby/setup-ruby@v1
        with:
          bundler-cache: true
          ruby-version: ${{matrix.ruby}}
      - run: bundle exec rubocop .
      - run: bundle exec rspec
      - run: bundle exec rake codecov:upload
```

This might work @jodosha